welcomebot.py

```diff
@@ -187,24 +187,31 @@ def perform_image_search():
     elif message.content.startswith("//search"):
         print "Is search request"
         if len(message.content.split()) == 1:
-            message.message.reply("Can't search, No terms given")
+            message.message.reply("Can't search, no terms given")
         else:
             def perform_google_search():
                 search_term = "+".join(message.content.split()[1:])
-                search_results = google_search.google_search(search_term)
-                print search_results
-                if search_results is False:
-                    print "No Results"
+                try:
+                    results, moreResultsUrl = google_search.google_search(search_term)
+                    print results
+                except TypeError:
```

Comment:
In the time of writing this piece of code, I was not sure about it, so I have tried the following:

```bash
kubo2@kubo2:~/WelcomeBot$ python -c 'a, b = False'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: 'bool' object is not iterable
kubo2@kubo2:~/WelcomeBot$ python --version
Python 2.7.6
```

The same it behaves when feeding up `a` and `b` from a function return value. You have to realize, that when you're doing `return False`, you're **not returning a tuple** (which is iterable) with not enough values, you're just simply returning a non-iterable `bool` object of value `False`.